aws/logs_monitoring/tools/Dockerfile_bundle

```diff
@@ -19,4 +19,3 @@ RUN rm -rf ./botocore*
 # like `os.execl`, which cause security scans to fail for certain customers.
 # These files are not directly used by the Forwarder.
 RUN rm ./ddtrace/commands/ddtrace_run.py
-RUN rm ./decorator.py
```

Comment:
why this change ?

Reply:
file not found error was thrown during ITests, it seems it was added during a dependency version bump which created this file.
Now, it is not created, so rm is failing